pages/common/plantuml.md

```diff
@@ -19,6 +19,14 @@
 
 `plantuml -o {{path/to/output}} {{diagram.puml}}`
 
+- Render a diagram without storing diagram's source code (stored by default without `-nometadata`):
```

Comment:
```suggestion
- Render a diagram without storing the diagram's source code (Note: It's stored by default when the `-nometadata` option isn't specified):
```

Reply:
This is much better, thanks for your revision. I was a bit worried about making the sentence too long, so I tried to keep it short. But if that's okay, what you wrote is much better and clear.